bin/generate-zbm

```diff
@@ -116,14 +116,16 @@ if ( ( defined $runConf{kernel} ) and ( length $runConf{kernel} ) ) {
 $runConf{bootdir} = dirname( $runConf{kernel} );
 ( $runConf{kernel_prefix}, $runConf{kernel_version} ) = split( '-', basename( $runConf{kernel} ) );
 
-# We always need to create an initramfs
-printf "Creating ZFS Boot Menu %s, with %s %s\n", $runConf{version}, $runConf{kernel_prefix}, $runConf{kernel_version};
+# We need to create an initramfs for all cases other than unifiedEFI
```

Comment:
If you add another type of output (besides component and unified), we'd need to add another case here. Do you think there's another way to write this conditional that is more general?